frontend/src/components/Card.jsx

```diff
@@ -0,0 +1,33 @@
+const Card = ({data}) => {
```

Comment:
This is specifically for event card view right? I would 1) put event in the component name, and 2) instead of using {data}, give props definition {name, location, imageUrl, score, numReviews}. That way you can make sure that the usage of this component would have all the required fields.

Reply:
To do

Reply:
Done